File: applied/4_dancing_queen/answer/1.java
// 1つの数字を入力して、その回数だけ"Hello world!"と出力しよう

import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        int input = scan.nextInt();
        
        for (int i=0; i<input; i++) {
            System.out.printf("Hello world!\n");
        }
    }
}

File: applied/4_dancing_queen/answer/1.c
// 1つの数字を入力して、その回数だけ"Hello world!"と出力しよう

#include <stdio.h>
int main(void){
    int i, input;
    scanf("%d", &input);
    
    for (i=0; i<input; i++) {
        printf("Hello world!\n");
    }
}
